docs/models.md

```diff
@@ -199,10 +199,14 @@ You can then use [`GeminiModel`][pydantic_ai.models.gemini.GeminiModel] by name:
 ```python {title="gemini_model_by_name.py"}
 from pydantic_ai import Agent
 
-agent = Agent('gemini-1.5-flash')
+agent = Agent('google-gla:gemini-1.5-flash')
 ...
 ```
 
+!!! note
+    The `google-gla` provider prefix represents the "google generative language api" for `GeminiModel`s.
```

Comment:
I think we need a link here to `https://ai.google.dev/api/all-methods` and Vertex AI here.